src/Git/CleanBuildDir.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\AutomaticReleases\Git;
+
+use Symfony\Component\Process\Process;
+
+final class CleanBuildDir
+{
+    public function __invoke(string $buildDir) : void
+    {
+        (new Process(['rm', '-rf', $buildDir]))
```

Comment:
At this point, we need to assert that `$buildDir` is a directory, before performing something this scary :)